diff -urN glom-1.6.13/configure.in glom-1.6.13.new/configure.in
--- glom-1.6.13/configure.in	2008-04-05 00:22:56.000000000 +0200
+++ glom-1.6.13.new/configure.in	2008-04-07 13:29:21.000000000 +0200
@@ -128,15 +128,6 @@
 AC_SUBST(POSTGRES_UTILS_PATH)
 AC_DEFINE_UNQUOTED([POSTGRES_UTILS_PATH], ["${POSTGRES_UTILS_PATH}"], [Path to the postgres utilities, such as postmaster.])
 
-if test $enable_client_only != yes; then
-	# Check that the supplied (or default) path really contains the postgres utilities:
-	AC_CHECK_PROG(HAVE_POSTMASTER, postmaster, yes, no, ${POSTGRES_UTILS_PATH})
-	if test "x$HAVE_POSTMASTER" = "xno"; then
-	AC_MSG_ERROR([The postgres utilities could not be found in the path: ${POSTGRES_UTILS_PATH}. They are needed for self-hosting of Glom databases. Please make sure that Postgres is installed, and provide the correct path with the --with-postgres-utils option.])
-	fi
-fi
-
-
 # Allow use of gnome-doc-utils to be disabled for platforms (For instance, Maemo) that don't have it:
 AC_ARG_ENABLE([doc-utils],
 	AC_HELP_STRING([--enable-doc-utils],
