diff -urN glom-1.8.5/configure.in glom-1.8.5.new/configure.in
--- glom-1.8.5/configure.in	2009-01-04 17:59:33.000000000 +0100
+++ glom-1.8.5.new/configure.in	2009-02-11 11:20:00.000000000 +0100
@@ -151,14 +151,6 @@
 
 	AC_SUBST(POSTGRES_UTILS_PATH)
 	AC_DEFINE_UNQUOTED([POSTGRES_UTILS_PATH], ["${POSTGRES_UTILS_PATH}"], [Path to the postgres utilities, such as postmaster.])
-
-	if test $enable_client_only != yes; then
-		# Check that the supplied (or default) path really contains the postgres utilities:
-		AC_CHECK_PROG(HAVE_POSTMASTER, postmaster, yes, no, ${POSTGRES_UTILS_PATH})
-		if test "x$HAVE_POSTMASTER" = "xno"; then
-		AC_MSG_ERROR([The postgres utilities could not be found in the path: ${POSTGRES_UTILS_PATH}. They are needed for self-hosting of Glom databases. Please make sure that Postgres is installed, and provide the correct path with the --with-postgres-utils option.])
-		fi
-	fi
 fi
 
 
