--- glom-1.32.0/glom/libglom/glom.pc.in.orig	2019-09-27 19:09:30.000000000 +0200
+++ glom-1.32.0/glom/libglom/glom.pc.in	2019-11-07 16:21:54.589338076 +0100
@@ -8,7 +8,7 @@
 
 Name: libglom
 Description: Library for accessing Glom documents and data structures.
-Requires: glibmm-2.4 giomm-2.4 libgdamm-5.0 libxml++-2.6 libxslt
+Requires: glibmm-2.4 giomm-2.4 libgdamm-5.0 libxml++-3.0 libxslt
 Version: @PACKAGE_VERSION@
 Libs: -L${libdir} -lglom-@GLOM_ABI_VERSION@
 Cflags: -I${includedir}/glom-@GLOM_ABI_VERSION@
